.test_env_config_fact_store.yaml

```diff
@@ -0,0 +1,9 @@
+STORAGE_BACKEND: "local"
```

Comment:
How is this file `.test_env_config_fact_store.yaml` used? Can't we re-use the exisiting `.test_env_config.yaml`?

Reply:
I think we should keep this additional file. Its an example of config that someone would use if say they wanted to run fact store locally and have anomaly detector work with it. I think we should keep it.

Reply:
ok. same, add the way to execute this to the README.md 
or, even better, add a `Makefile` that would execute this and other ways to run tests etc.
Just like here: https://github.com/AICoE/prometheus-flatliner/blob/master/Makefile#L61

Reply:
Yup I added a make file to this PR.